Xamarin.Forms.Xaml/ApplyPropertiesVisitor.cs

```diff
@@ -95,20 +97,23 @@ public void Visit(ElementNode node, INode parentNode)
 				parentElement = parentNode as IElementNode;
 			}
 
-			var value = Values[node];
+			if (!Values.TryGetValue(node, out var value) && Context.ExceptionHandler != null)
```

Comment:
Are we happy with the `var s0 = Values[parentNode];` in the chunk above?

Reply:
TryGetPropertyName will probably fail on unregistered parentNode, so it looks safe. again, I want this code to keep throwing on anything going wrong